### ISSUE TITLE
[TW-1103] display .xpm icons

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1138]
_David Patrick on 2010-02-09T10:14:17Z says:_ 

 These are ascii icons from long ago, seen in ancient Windows systems, old window managers and occasionally adorning emacs buffers. This occurred to me on seeing emacs orgmode screenshots featuring tiny icons ! like checkboxes, bullets and all sorts of things. I don't know if this is even remotely possible for task, and it's not particularly important, but I do know that tiny icons rendered via cli or interactive display, would indeed offer very strong visual clues.. and cute !

http://en.wikipedia.org/wiki/X_PixMap

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-02-09T10:14:17Z
Modified: 2014-02-09T02:18:07Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-02-10T22:36:27Z says:_ 

 David, see attached picture.  Is that what you mean?

**Comment by taskwarrior** [comment]
_David Patrick on 2010-02-11T00:21:17Z says:_ 

 yes

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-02-11T07:07:44Z says:_ 

 Good news, David, that attached picture is real.  This is already supported.

If you go find some funky unicode characters (there are lots of web sites dedicated to listing them), you can select them and paste them in the command line, and task will accept it.  So will Redmine: ? ? ? ? ? ? ? ? ? ? ? ? ? ?.  In the example, I used them as tags, and in descriptions.  You can probably use them anywhere.  Here is roughly what I did:

<pre>
$ task add Call Bob about the meeting +?
$ task add Book ? flight to conference
$ task add Check out the caffe on Harbord +?
</pre>

That last one is listed as "hot beverage", but it's hard to see at any size.  You'll appreciate why I needed to use that one.  And that telephone character (I found two of them) is becoming anachronistic.  It might also help that I have this in my environment:

<pre>
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
</pre>

The downside to using unicode characters is that some of them are more than one character wide in the display, despite using a fixed-width font.  While this works, task is not smart enough to know this, so it tends to throw off the output table alignment.  You may notice in the picture that the tasks don't line up properly - that's going to be addressed in task 2.0, with luck.

**Comment by pbeckingham** [comment]
![tw-1103_unicode](https://user-images.githubusercontent.com/7963/36643357-054dca1c-1a18-11e8-9d0f-d98438159ee9.png)
